iwyu/rules.imp.orig

```diff
@@ -0,0 +1,17 @@
+[
+    { include: ["<vcruntime_string.h>", "private", "<string.h>", "public"] },
+    { include: [<basetsd.h>, "private", "\"windows.h\"", "public"] },
+    { include: [<minwindef.h>, "private", "\"windows.h\"", "public"] },
+<<<<<<< HEAD
+    { include: [<synchapi.h>, "private", "\"windows.h\"", "public"] },
+    { include: [<winerror.h>, "private", "\"windows.h\"", "public"] },
+    { include: [<processthreadsapi.h>, "private", "\"windows.h\"", "public"] },
```

Comment:
>[< [](start = 15, length = 2)

file accidentally included

Reply:
possibly missing .gitignore for .orig files

---
In reply to: [561233280](https://github.com/Azure/c-build-tools/pull/56#discussion_r561233280) [](ancestors = 561233280)